distributions/aws/examples/storage/efs/auto-efs-setup.py

```diff
@@ -537,6 +538,14 @@ def footer():
     print("                      EFS Setup Complete")
     print("=================================================================")
 
+def rand_name(prefix):
```

Comment:
We should go the other way. Modify this script to use it in the tests

Reply:
Do you mean "don't modify this script but modify usage in tests instead" as I mentioned above ?

Reply:
done, if I understood you correctly, can you confirm.